eng/common/scripts/ChangeLog-Operations.ps1

```diff
@@ -120,7 +120,15 @@ function Confirm-ChangeLogEntry {
     else {
       $status = $changeLogEntry.ReleaseStatus.Trim().Trim("()")
       try {
-        [DateTime]$status
+        $releaseDate = [DateTime]$status
+        if ($status -ne ($releaseDate.ToString($CHANGELOG_DATE_FORMAT)))
+        {
+          throw "Date must be in the format $($CHANGELOG_DATE_FORMAT)"
```

Comment:
We should use a LogError message here also as below because the message will get lost in the catch block